cli/src/main/java/org/jboss/as/cli/impl/CommandExecutor.java

```diff
@@ -633,7 +633,7 @@ public ModelNode execute(Operation op, String description) throws CommandLineExc
     }
 
     private final CommandContext ctx;
-    private final ExecutorService executorService = Executors.newSingleThreadExecutor();
+    private final ExecutorService executorService = Executors.newCachedThreadPool();
```

Comment:
This isn't necessary for this PR -- just a thought...

Any pools we create should have a ThreadFactory so the threads get a meaningful name.

This isn't such a big deal with a standalone CLI, but given embed-server makes embedding the CLI more useful, it's a good practice.